### ISSUE TITLE
Warehouse Schedule suffers from some usability problems

### ISSUE THREAD

**Issue by vqmarkman** [sundeck-io/OpsCenter#436]
* slow loading and saving schedules (maybe it's my perception, but that's my overall impression )
* after new schedule is created and save button is pressed user is taken to a completely different tab with the report.
  (I believe it is something that we know about and we think it is streamlit ? see #40   )
* worse, after new schedule is created, check button of "Enable schedule" is unchecked again! Drove me crazy ...

**Comment by joshelser** [comment]
> slow loading and saving schedules (maybe it's my perception, but that's my overall impression )

On our developer account, building the cost/usage summary over all of the warehouses we currently have (the data in the Filter dropdown) takes ~9seconds on an X-Small warehouse. Using a Small or Medium (I forget which I tried) decreased this to ~3 seconds. I would assume that is what you are noticing.

I didn't think it was prudent to optimize this lookup right now.

> after new schedule is created and save button is pressed user is taken to a completely different tab with the report.
(I believe it is something that we know about and we think it is streamlit)

I am 95% certain that Streamlit does not give you a way to specify which tab is active (I went looking for this and remember reading some blog post). I'll double check.
